pkg/controller/controller.go

```diff
@@ -15,9 +15,10 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
-//Controller is an interface of controller
+// Controller is an interface of controller
 type Controller interface {
-	CertsGet(devUUID uuid.UUID) (out string, err error)
+	DevCertsGet(dev *device.Ctx) ([]byte, error)
```

Comment:
Should we implement this new Functions in every struct which implements Controller?

Reply:
afaik Adam is the only one implementing Controller and it already implements this function... I think the code wouldn't compile if it wasn't implemented for all structs...